Sources/MenuTransitioningDelegate.swift

```diff
@@ -21,7 +21,7 @@ import Foundation
 /**
  Delegate of menu transitioning actions.
  */
-final class MenuTransitioningDelegate: NSObject {
+public final class MenuTransitioningDelegate: NSObject {
```

Comment:
@truemetal At the moment, we consider that this should not be accessible from client code. Have you run into a case when you need to use this?